modules/multipath/data/org.freedesktop.UDisks2.Multipath.xml

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE node PUBLIC
+"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+
+<!--
+ Copyright (C) 2015 Gris Ge <fge@redhat.com>
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General
+ Public License along with this library; if not, write to the
+ Free Software Foundation, Inc., 59 Temple Place, Suite 330,
+ Boston, MA 02111-1307, USA.
+-->
+
+  <!-- ******************************************************************* -->
+
+  <!--
+    org.freedesktop.UDisks2.Drive.Multipath:
+    @short_description: Multipath interface exported on disk objects
+  -->
+  <interface name="org.freedesktop.UDisks2.Drive.Multipath">
+    <!-- Multipath:
+        If the disk drive is multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
+    <property name="Multipath" type="o" access="read"/>
+  </interface>
+  <!--
+    org.freedesktop.UDisks2.Block.Multipath:
+    @short_description: Multipath interface exported on block objects
+  -->
+  <interface name="org.freedesktop.UDisks2.Block.Multipath">
+    <!-- Path:
+        If the block is a multipath device slave, this is set to the
+        org.freedesktop.UDisks2.Multipath.Path object that it
+        corresponds to. It is '/' otherwise.
+    -->
+    <property name="Path" type="o" access="read"/>
+    <!-- Multipath:
+        If block is(or part of) multipath device, this is set to the
+        org.freedesktop.UDisks2.Multipath object that it corresponds to. It is
+        '/' otherwise.
+    -->
+    <property name="Multipath" type="o" access="read"/>
```

Comment:
I can't help but think that we only need one of these: either `Path` or `Multipath`, since either can be obtained given the other. I realize having both is convenient, but it's also causing the namespace to become a bit cluttered.